--- devhelp-3.34.0/devhelp/dh-book-list-builder.c.orig	2019-11-13 16:42:36.696750189 +0100
+++ devhelp-3.34.0/devhelp/dh-book-list-builder.c	2019-11-13 16:42:40.750061563 +0100
@@ -140,6 +140,10 @@
         add_book_list_directory (builder, dir);
         g_free (dir);
 
+        dir = g_build_filename (data_dir, "doc", "gtk-doc", "html", NULL);
+        add_book_list_directory (builder, dir);
+        g_free (dir);
+
         dir = g_build_filename (data_dir, "devhelp", "books", NULL);
         add_book_list_directory (builder, dir);
         g_free (dir);
