diff -urN devhelp-0.21/m4/gecko.m4 devhelp-0.21.new/m4/gecko.m4
--- devhelp-0.21/m4/gecko.m4	2008-09-08 09:29:56.000000000 +0200
+++ devhelp-0.21.new/m4/gecko.m4	2008-10-10 09:48:12.000000000 +0200
@@ -116,7 +116,7 @@
 	_GECKO_PREFIX="`$PKG_CONFIG --variable=prefix ${gecko_cv_gecko}-xpcom`"
 	_GECKO_NSPR=no # XXX asac: this is currently a blind guess and should be a AC test
 else
-	_GECKO_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir ${gecko_cv_gecko}`/unstable"
+	_GECKO_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir ${gecko_cv_gecko}`"
 	_GECKO_CFLAGS="`$PKG_CONFIG --cflags ${gecko_cv_gecko}` `$PKG_CONFIG --cflags ${gecko_cv_gecko}-unstable`"
 	_GECKO_LIBDIR="`$PKG_CONFIG --variable=sdkdir ${gecko_cv_gecko}`/bin"
 	_GECKO_HOME=$with_gecko_home
