diff -urN aa/devhelp-0.9.3/configure.in devhelp-0.9.3/configure.in
--- aa/devhelp-0.9.3/configure.in	2004-10-21 23:40:35.000000000 +0200
+++ devhelp-0.9.3/configure.in	2004-11-01 19:02:40.012992080 +0100
@@ -84,7 +84,7 @@
 MOZILLA_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir $MOZILLA-gtkmozembed`"
 AC_SUBST(MOZILLA_INCLUDE_ROOT)
 
-MOZILLA_HOME="`$PKG_CONFIG --variable=libdir $MOZILLA-gtkmozembed`"
+MOZILLA_HOME="`$PKG_CONFIG --variable=libdir $MOZILLA-plugin`"
 AC_SUBST(MOZILLA_HOME)
 
 MOZILLA_PREFIX="`$PKG_CONFIG --variable=prefix $MOZILLA-gtkmozembed`"
