diff -urN devhelp-0.13/devhelp.desktop.in.in devhelp-0.13.new/devhelp.desktop.in.in
--- devhelp-0.13/devhelp.desktop.in.in	2007-01-10 16:40:41.000000000 +0100
+++ devhelp-0.13.new/devhelp.desktop.in.in	2007-02-17 14:26:33.000000000 +0100
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 Icon=devhelp.png
-Categories=GNOME;GTK;Application;Development;
+Categories=GTK;GNOME;Development;
 StartupNotify=true
 MimeType=application/x-devhelp;
 X-GNOME-Bugzilla-Bugzilla=GNOME
