diff -urN devhelp-0.13/src/dh-base.c devhelp-0.13.new/src/dh-base.c
--- devhelp-0.13/src/dh-base.c	2007-02-07 14:41:04.000000000 +0100
+++ devhelp-0.13.new/src/dh-base.c	2007-02-07 16:05:31.000000000 +0100
@@ -229,6 +229,10 @@
 	dir = g_build_filename (data_dir, "gtk-doc", "html", NULL);
 	base_add_books (base, dir);
 	g_free (dir);
+	
+	dir = g_build_filename (data_dir, "doc", "gtk-doc", "html", NULL);
+	base_add_books (base, dir);
+	g_free (dir);
 
 	dir = g_build_filename (data_dir, "devhelp", "books", NULL);
 	base_add_books (base, dir);
