diff -urN devhelp-0.22/src/dh-base.c devhelp-0.22.new/src/dh-base.c
--- devhelp-0.22/src/dh-base.c	2008-10-11 00:00:18.000000000 +0200
+++ devhelp-0.22.new/src/dh-base.c	2008-12-03 20:27:27.000000000 +0100
@@ -210,6 +210,10 @@
         dir = g_build_filename (data_dir, "gtk-doc", "html", NULL);
         base_add_books (base, dir);
         g_free (dir);
+        
+        dir = g_build_filename (data_dir, "doc", "gtk-doc", "html", NULL);
+        base_add_books (base, dir);
+        g_free (dir);
 
         dir = g_build_filename (data_dir, "devhelp", "books", NULL);
         base_add_books (base, dir);
